Octokit.Tests/Clients/StarredClientTests.cs

```diff
@@ -346,19 +345,31 @@ public class TheGetAllForUserMethod
         public class TheGetAllStargazersForRepoMethod
         {
             [Fact]
-            public void RequestsCorrectUrl()
+            public async Task RequestsCorrectUrl()
```

Comment:
While it's technically correct to `await` in these tests, I've not seen this affect the test behaviour - as we're interested in what was invoked, rather than the result returned. Although, saying it out loud like that makes me feel like we're just asking for some neat race condition to come along and cause us issues...